ArcGISRuntimeSDKQt_CppSamples/Geometry/ClipGeometry/README.md

```diff
@@ -0,0 +1,26 @@
+# Clip Geometry
+
+This sample demonstrates how to clip a geometry with an envelope using the GeometryEngine.
+
+![](screenshot.png)
+
+## How to use the sample
+Click or tap the 'Clip' button to display the clipping results of the blue graphic and the red outlines.
+
+## How it works
+1. Create at least three  `graphicsOverlays`, one with a reference graphic, one (or more) with an overlapping envelope-based graphic, and one empty one to contain the result
+2. Add each `graphicsOverlay` to the `mapView`
+2. Create a variable that stores the result of the static function `GeometryEngine::clip', which accepts a reference `geometry` and an `envelope`
```

Comment:
a few typos:
- this is step 3 (says 2)
- after ::clip, there should be a ` but instead there is a '